### PR TITLE
Implement endpoints for getting and setting display orientation

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
I'm not sure about the in/out parameters format, just copied these from UIA2 server

### CODE REVIEW

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityFinder.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import android.view.ViewGroup;
+
+public class ActivityFinder {
+    public static Activity extractFrom(View view) {
+        Activity result = getActivity(view.getContext());
+        if (result == null && (view instanceof ViewGroup)) {
+            ViewGroup v = (ViewGroup) view;
+            int c = v.getChildCount();
+            for (int i = 0; i < c && result == null; ++i) {
+                result = getActivity(v.getChildAt(i).getContext());
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityFinder.java]: it shouldn't break, it's correct

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityFinder.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            }
+        }
+        return result;
+    }
+
+    private static Activity getActivity(Context ctx) {
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityFinder.java]: ok, ok, let's make the skynet happy

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityFinder.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityFinder.java]: pay attention to break condition: `i < c && result == null`

[the remark this answers is not in the document]
Comment (dpgraham) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityFinder.java]: Ahh yes.

**Pull request merged**